shippable.yml

```diff
@@ -9,15 +9,15 @@ node_js:
   - 0.10.28
 
 addons:
-  firefox: 29.0
+  firefox: 47.0.1
 
 install:
   # install global dependencies
   - npm install bower grunt-cli -g
 
   # install dependencies for both client and server side
   - (cd client ; npm install)
-  - (cd client ; bower install)
+  - (cd client ; bower install --allow-root)
```

Comment:
I'm not thrilled with this either.  Apparently, shippable's container runs these commands with root access to the container.  Recent versions of bower have started throwing an exception under these conditions (sample pasted below).

The common workaround, both in the shippable community and bower itself, seems to be the use of this flag to signal bower that we're willing to let it run with superuser permissions.  Given that this is a short-lived, virtualized and sandboxed container, this doesn't seem to be _too_ awful even though it is less than ideal.

I'm still looking into getting the container to run these commands without elevated access.



(cd client ; bower install) 0s  
[{
  "code": "ESUDO",
  "details": "Since bower is a user command, there is no need to execute it with superuser permissions.\nIf you're having permission errors when using bower without sudo, please spend a few minutes learning more about how your system should work and make any necessary repairs.\n\nhttp://www.joyent.com/blog/installing-node-and-npm\nhttps://gist.github.com/isaacs/579814\n\nYou can however run a command with sudo using --allow-root option",
  "id": "ESUDO",
  "level": "error",
  "data": {},
  "message": "Cannot be run with sudo",
  "stacktrace": "Error: Cannot be run with sudo\n    at createError (/root/.nvm/v0.10.28/lib/node_modules/bower/lib/util/createError.js:4:15)\n    at rootCheck (/root/.nvm/v0.10.28/lib/node_modules/bower/lib/util/rootCheck.js:26:24)\n    at Object.<anonymous> (/root/.nvm/v0.10.28/lib/node_modules/bower/lib/bin/bower.js:33:1)\n    at Module._compile (module.js:456:26)\n    at Object.Module._extensions..js (module.js:474:10)\n    at Module.load (module.js:356:32)\n    at Function.Module._load (module.js:312:12)\n    at Module.require (module.js:364:17)\n    at require (module.js:380:17)\n    at Object.<anonymous> (/root/.nvm/v0.10.28/lib/node_modules/bower/bin/bower:3:1)"
}]